data/posts/split-route-modules.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Split Route Modules
+summary: React Router v7.2's framework mode now supports automatic code splitting of route modules
+date: 2025-02-27
+image: /blog-images/headers/react-router-v7.jpg
+ogImage: /blog-images/headers/react-router-v7.jpg
+imageAlt: "The React Router logo"
+imageDisableOverlay: true
+authors:
+  - Mark Dalgleish
+---
+
+With the release of [React Router v7.2.0](https://github.com/remix-run/react-router/releases/tag/react-router%407.2.0), we’ve introduced a new opt-in framework feature called Split Route Modules. In this post we’ll explore the performance problem that Split Route Modules solves, how it works, and how to use it today.
```

Comment:
We should probably call out that it's currently unstable and not for production use?  Or are we planning to publish this once it's stabilized?

Reply:
My 2 cents: I'd prefer us to not wait until the feature is stabilized. Blogs are a good place to talk about features no matter where they are in the process. Docs are where the most up to date information can always been found.

I'm down for a disclaimer saying this is currently an unstable API. We can either update/remove when it goes stable.

Happy to hear Mark's thoughts though

Reply:
I've added a note to try and cover this, I'd appreciate your thoughts on it.